src/seedu/addressbook/formatter/Formatter.java

```diff
@@ -147,4 +147,24 @@ public static String formatAssessments(List<? extends Assessment> assessments) {
         }
         return format(asIndexedList(formattedAssessments));
     }
+
+    /**
+     * Formats a string for an exam to be printed
+     */
+    public static String getPrintableExam(String examName, String subjectName, String examDate, String examStartTime,
+                                          String examEndTime, String examDetails, int takers, boolean isPrivate) {
+        final String examFormat = "%1$sExam: %2$s %3$s %4$s %5$s %6$s %7$s. %9$s: %8$d";
+        String takerWord;
+        if (takers == 1) {
+            takerWord = "Taker";
+        } else {
+            takerWord = "Takers";
+        }
```

Comment:
Very small nitpick, takerWord can be final

Reply:
Resolved.